TST: type / coercion issues, xref #10976

xref #10976 broke some tests on windows (these prob just need some ``_ensure_int64(..)`` before passing the pointers in, the ``test_rsample_group_info/test_series_groupby_nunique`` are from a previous issue; these just need ``np.array(...., dtype='int64')`` in the comparison functions (as windows makes these ``np.int32`` by default)

cc @behzadnouri  

```
[pandas] C:\Users\Jeff Reback\pandas>
[pandas] C:\Users\Jeff Reback\pandas>nosetests --exe -A "not slow and not network and not disabled" build\lib.win-amd64-2.7\pandas
C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\io\data.py:33: FutureWarning:
The pandas.io.data module is moved to a separate package (pandas-datareader) and will be removed from pandas in a future version.
After installing the pandas-datareader package (https://github.com/pydata/pandas-datareader), you can change the import ``from pandas.io import data, wb`` to ``from pandas_datareader import data, wb``
.
  FutureWarning)
..SS..SS..SS..SS..SS..SS..SSSSSSSSSS..SS..SS..SSSS...........SS..SS..SS..SS..SS..SS..SS................SSSSSSSSSSSSSSSS..SS..SS...............S....S........S......SSSSSSSSSSSSSSSSSSSSS.SSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS.S..................................................................................................................................S..................
...............................S............................................S...............................................................................................................S...........
........S..........................S...................................................................S....................................................S............S..............................
......................................................................................................................S.....SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS.
....................................................S..........................................................................................S........................................................
......................................................S............S.....................................S.....................SSSSSSSSSSSSS..S.SSSSSSSSSS............S.....S...........................
.............S.....SSSSSSSSSSSSSSSSSSSSSSSSSSS..........................................................................................................................................................
.......................S.....................................................S..........................................................................................................................
............................S..S........................................................................................................................................................................
.........S..................................................S...S.......................................................................................................................................
.....................................................................S...E.E.E....E......FE.EFE....EEEE..............E.............EE........E..............EF.....E....................................
..............................................................................................................................................................................................E.........
..................................................................................................................................................S.....................................................
.....................................................................................................................................................C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7
\pandas\core\algorithms.py:149: RuntimeWarning: tp_compare didn't return -1 or -2 for exception
  sorter = uniques.argsort()
C:\Miniconda\envs\pandas\lib\site-packages\numpy\core\fromnumeric.py:799: RuntimeWarning: tp_compare didn't return -1 or -2 for exception
  a.sort(axis, kind, order)
...........................................................................................................................................................................................S............
........................................................................................................................................................................................................
....................................................................................................................S.SSSSSC:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tests\test_frame.p
y:7709: RuntimeWarning: Cannot compare type 'Timestamp' with type 'str', sort order is undefined for incomparable objects
  colcorr = a.corrwith(b, axis=0)
.....................................................................................................................................................................................S..................
...................................S.S.............................................................................................................................................S....................
...................................................................................SSS...SSSS.S.S.S.S.S.S.SSS.SSS.S.S.S.S.S.S.S...S.S.SSSSSSSSS..................S......S..SS..S.........SSSSS..........
....S..................SS......SSSSS.S....S...........SSSS..............................................................................................................................................
.....................F.............................................................F..........F.........................................................................................................
........................................................................................................................................................................................................
........................................................................................................................................................................................................
...............................................................................................................................................................S........................................
........................................................................................................................................................................................................
........................................................................................................................................................................................................
........................................................................................................................................................................................................
........................................................................................................................................................................................................
........................................................................................................................................................................................................
........................................................................................................................................................................................................
........................................................................................................................................................................................................
........................................................................................................................................................................................................
........................................................................................................................................................................................................
...........................................................................................................S..............................................................................S.S....S......
...SS............................................................................................................S.....SS.......................S...S.............SSS.........S...........SS............
...SSS....SSSS..................................................SSSSS..SS..........................................................................................SS...................................
..................................................S..........................S................................SS....................................................S...................................
.......................................SS..........................................................................................S.......S...............................................S.....E......
..
======================================================================
ERROR: test_custom_grouper (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 54, in test_custom_grouper
    g._cython_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1129, in _cython_agg_general
    result, names = self.grouper.aggregate(obj.values, how)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1566, in aggregate
    result = self._aggregate(result, counts, values, agg_func, is_numeric)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1611, in _aggregate
    agg_func(result, counts, values, comp_ids)
  File "pandas\src\generated.pyx", line 6743, in pandas.algos.group_add_float64 (pandas\algos.c:118485)
    def group_add_float64(ndarray[float64_t, ndim=2] out,
ValueError: Buffer dtype mismatch, expected 'int64_t' but got 'long'

======================================================================
ERROR: test_how_lambda_functions (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 850, in test_how_lambda_functions
    bar_exp = ts.resample('M', how='std')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 82, in resample
    rs = self._resample_timestamps()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 287, in _resample_timestamps
    result = grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 766, in std
    return np.sqrt(self.var(ddof=ddof))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 775, in var
    return self._cython_agg_general('var')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1129, in _cython_agg_general
    result, names = self.grouper.aggregate(obj.values, how)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1566, in aggregate
    result = self._aggregate(result, counts, values, agg_func, is_numeric)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1611, in _aggregate
    agg_func(result, counts, values, comp_ids)
  File "pandas\src\generated.pyx", line 6988, in pandas.algos.group_var_float64 (pandas\algos.c:121664)
    def group_var_float64(ndarray[float64_t, ndim=2] out,
ValueError: Buffer dtype mismatch, expected 'int64_t' but got 'long'

======================================================================
ERROR: test_ohlc_5min (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 504, in test_ohlc_5min
    label='right')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 82, in resample
    rs = self._resample_timestamps()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 287, in _resample_timestamps
    result = grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 816, in ohlc
    lambda x: x._cython_agg_general('ohlc'))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2689, in _apply_to_column_groupbys
    return func(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 816, in <lambda>
    lambda x: x._cython_agg_general('ohlc'))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1129, in _cython_agg_general
    result, names = self.grouper.aggregate(obj.values, how)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1566, in aggregate
    result = self._aggregate(result, counts, values, agg_func, is_numeric)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1611, in _aggregate
    agg_func(result, counts, values, comp_ids)
  File "pandas\src\generated.pyx", line 7195, in pandas.algos.group_ohlc_float64 (pandas\algos.c:124745)
    def group_ohlc_float64(ndarray[float64_t, ndim=2] out,
ValueError: Buffer dtype mismatch, expected 'int64_t' but got 'long'

======================================================================
ERROR: test_resample_axis1 (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 539, in test_resample_axis1
    result = df.resample('M', axis=1)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 82, in resample
    rs = self._resample_timestamps()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 287, in _resample_timestamps
    result = grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2786, in aggregate
    return getattr(self, arg)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1147, in _python_agg_general
    result, counts = self.grouper.agg_series(obj, f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1862, in agg_series
    return grouper.get_result()
  File "pandas\src\reduce.pyx", line 274, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:34306)
    raise
  File "pandas\src\reduce.pyx", line 260, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:34160)
    res = self.f(cached_typ)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1141, in <lambda>
    f = lambda x: func(x, *args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 737, in <lambda>
    f = lambda x: x.mean(axis=self.axis)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 4509, in stat_func
    skipna=skipna, numeric_only=numeric_only)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\series.py", line 2111, in _reduce
    self._get_axis_number(axis)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 312, in _get_axis_number
    .format(axis, type(self)))
ValueError: No axis named 1 for object type <class 'pandas.core.series.Series'>

======================================================================
ERROR: test_resample_empty (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 688, in test_resample_empty
    empty_series.resample('d', how)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 82, in resample
    rs = self._resample_timestamps()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 287, in _resample_timestamps
    result = grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1147, in _python_agg_general
    result, counts = self.grouper.agg_series(obj, f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1862, in agg_series
    return grouper.get_result()
  File "pandas\src\reduce.pyx", line 227, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:33564)
    counts[0] = self.bins[0]
IndexError: index 0 is out of bounds for axis 0 with size 0

======================================================================
ERROR: test_resample_frame_basic (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 322, in test_resample_frame_basic
    g._cython_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2711, in _cython_agg_general
    new_items, new_blocks = self._cython_agg_blocks(how, numeric_only=numeric_only)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2747, in _cython_agg_blocks
    result, _ = self.grouper.aggregate(block.values, how, axis=agg_axis)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1566, in aggregate
    result = self._aggregate(result, counts, values, agg_func, is_numeric)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1611, in _aggregate
    agg_func(result, counts, values, comp_ids)
  File "pandas\src\generated.pyx", line 6743, in pandas.algos.group_add_float64 (pandas\algos.c:118485)
    def group_add_float64(ndarray[float64_t, ndim=2] out,
ValueError: Buffer dtype mismatch, expected 'int64_t' but got 'long'

======================================================================
ERROR: test_resample_how (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 131, in test_resample_how
    closed='right', label='right')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 82, in resample
    rs = self._resample_timestamps()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 287, in _resample_timestamps
    result = grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 766, in std
    return np.sqrt(self.var(ddof=ddof))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 775, in var
    return self._cython_agg_general('var')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1129, in _cython_agg_general
    result, names = self.grouper.aggregate(obj.values, how)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1566, in aggregate
    result = self._aggregate(result, counts, values, agg_func, is_numeric)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1611, in _aggregate
    agg_func(result, counts, values, comp_ids)
  File "pandas\src\generated.pyx", line 6988, in pandas.algos.group_var_float64 (pandas\algos.c:121664)
    def group_var_float64(ndarray[float64_t, ndim=2] out,
ValueError: ("Buffer dtype mismatch, expected 'int64_t' but got 'long'", 'how=std')

======================================================================
ERROR: test_resample_ohlc (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 406, in test_resample_ohlc
    result = s.resample('5Min', how='ohlc')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 82, in resample
    rs = self._resample_timestamps()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 287, in _resample_timestamps
    result = grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 816, in ohlc
    lambda x: x._cython_agg_general('ohlc'))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2689, in _apply_to_column_groupbys
    return func(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 816, in <lambda>
    lambda x: x._cython_agg_general('ohlc'))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1129, in _cython_agg_general
    result, names = self.grouper.aggregate(obj.values, how)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1566, in aggregate
    result = self._aggregate(result, counts, values, agg_func, is_numeric)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1611, in _aggregate
    agg_func(result, counts, values, comp_ids)
  File "pandas\src\generated.pyx", line 7195, in pandas.algos.group_ohlc_float64 (pandas\algos.c:124745)
    def group_ohlc_float64(ndarray[float64_t, ndim=2] out,
ValueError: Buffer dtype mismatch, expected 'int64_t' but got 'long'

======================================================================
ERROR: test_resample_ohlc_dataframe (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 431, in test_resample_ohlc_dataframe
    res = df.resample('H', how='ohlc')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 82, in resample
    rs = self._resample_timestamps()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 315, in _resample_timestamps
    result = grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2786, in aggregate
    return getattr(self, arg)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 816, in ohlc
    lambda x: x._cython_agg_general('ohlc'))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 3460, in _apply_to_column_groupbys
    keys=self._selected_obj.columns, axis=1)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tools\merge.py", line 760, in concat
    copy=copy)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tools\merge.py", line 790, in __init__
    objs = list(objs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 3458, in <genexpr>
    (func(col_groupby) for _, col_groupby
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 816, in <lambda>
    lambda x: x._cython_agg_general('ohlc'))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1129, in _cython_agg_general
    result, names = self.grouper.aggregate(obj.values, how)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1566, in aggregate
    result = self._aggregate(result, counts, values, agg_func, is_numeric)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1611, in _aggregate
    agg_func(result, counts, values, comp_ids)
  File "pandas\src\generated.pyx", line 7195, in pandas.algos.group_ohlc_float64 (pandas\algos.c:124745)
    def group_ohlc_float64(ndarray[float64_t, ndim=2] out,
ValueError: Buffer dtype mismatch, expected 'int64_t' but got 'long'

======================================================================
ERROR: test_resample_panel (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 552, in test_resample_panel
    result = panel.resample('M', axis=1)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 82, in resample
    rs = self._resample_timestamps()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 287, in _resample_timestamps
    result = grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 3501, in aggregate
    return getattr(self, arg)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1145, in _python_agg_general
    for name, obj in self._iterate_slices():
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 3477, in _iterate_slices
    raise NotImplementedError("axis other than 0 is not supported")
NotImplementedError: axis other than 0 is not supported

======================================================================
ERROR: test_resample_panel_numpy (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 578, in test_resample_panel_numpy
    expected = panel.resample('M', how='mean', axis=1)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 82, in resample
    rs = self._resample_timestamps()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 287, in _resample_timestamps
    result = grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 3501, in aggregate
    return getattr(self, arg)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1145, in _python_agg_general
    for name, obj in self._iterate_slices():
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 3477, in _iterate_slices
    raise NotImplementedError("axis other than 0 is not supported")
NotImplementedError: axis other than 0 is not supported

======================================================================
ERROR: test_all_values_single_bin (pandas.tseries.tests.test_resample.TestResamplePeriodIndex)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 1411, in test_all_values_single_bin
    result = s.resample("A", how='mean')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 92, in resample
    rs = self._resample_periods()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 357, in _resample_periods
    return grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1147, in _python_agg_general
    result, counts = self.grouper.agg_series(obj, f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1862, in agg_series
    return grouper.get_result()
  File "pandas\src\reduce.pyx", line 274, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:34306)
    raise
  File "pandas\src\reduce.pyx", line 249, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:33847)
    cached_ityp = self.ityp(islider.buf)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 184, in __new__
    ordinal, freq = cls._from_arraylike(data, freq, tz)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 250, in _from_arraylike
    raise ValueError('freq not specified and cannot be '
ValueError: freq not specified and cannot be inferred from first element

======================================================================
ERROR: test_basic_downsample (pandas.tseries.tests.test_resample.TestResamplePeriodIndex)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 1063, in test_basic_downsample
    result = ts.resample('a-dec')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 92, in resample
    rs = self._resample_periods()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 357, in _resample_periods
    return grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1147, in _python_agg_general
    result, counts = self.grouper.agg_series(obj, f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1862, in agg_series
    return grouper.get_result()
  File "pandas\src\reduce.pyx", line 274, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:34306)
    raise
  File "pandas\src\reduce.pyx", line 249, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:33847)
    cached_ityp = self.ityp(islider.buf)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 184, in __new__
    ordinal, freq = cls._from_arraylike(data, freq, tz)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 250, in _from_arraylike
    raise ValueError('freq not specified and cannot be '
ValueError: freq not specified and cannot be inferred from first element

======================================================================
ERROR: test_basic_upsample (pandas.tseries.tests.test_resample.TestResamplePeriodIndex)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 1084, in test_basic_upsample
    result = ts.resample('a-dec')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 92, in resample
    rs = self._resample_periods()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 357, in _resample_periods
    return grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1147, in _python_agg_general
    result, counts = self.grouper.agg_series(obj, f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1862, in agg_series
    return grouper.get_result()
  File "pandas\src\reduce.pyx", line 274, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:34306)
    raise
  File "pandas\src\reduce.pyx", line 249, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:33847)
    cached_ityp = self.ityp(islider.buf)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 184, in __new__
    ordinal, freq = cls._from_arraylike(data, freq, tz)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 250, in _from_arraylike
    raise ValueError('freq not specified and cannot be '
ValueError: freq not specified and cannot be inferred from first element

======================================================================
ERROR: test_quarterly_resampling (pandas.tseries.tests.test_resample.TestResamplePeriodIndex)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 1344, in test_quarterly_resampling
    result = ts.resample('A')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 92, in resample
    rs = self._resample_periods()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 357, in _resample_periods
    return grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1147, in _python_agg_general
    result, counts = self.grouper.agg_series(obj, f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1862, in agg_series
    return grouper.get_result()
  File "pandas\src\reduce.pyx", line 274, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:34306)
    raise
  File "pandas\src\reduce.pyx", line 249, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:33847)
    cached_ityp = self.ityp(islider.buf)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 184, in __new__
    ordinal, freq = cls._from_arraylike(data, freq, tz)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 250, in _from_arraylike
    raise ValueError('freq not specified and cannot be '
ValueError: freq not specified and cannot be inferred from first element

======================================================================
ERROR: test_aggregate_normal (pandas.tseries.tests.test_resample.TestTimeGrouper)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 1552, in test_aggregate_normal
    dt_result = getattr(dt_grouped, func)()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 775, in var
    return self._cython_agg_general('var')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2711, in _cython_agg_general
    new_items, new_blocks = self._cython_agg_blocks(how, numeric_only=numeric_only)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2747, in _cython_agg_blocks
    result, _ = self.grouper.aggregate(block.values, how, axis=agg_axis)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1566, in aggregate
    result = self._aggregate(result, counts, values, agg_func, is_numeric)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1611, in _aggregate
    agg_func(result, counts, values, comp_ids)
  File "pandas\src\generated.pyx", line 6988, in pandas.algos.group_var_float64 (pandas\algos.c:121664)
    def group_var_float64(ndarray[float64_t, ndim=2] out,
ValueError: Buffer dtype mismatch, expected 'int64_t' but got 'long'

======================================================================
ERROR: test_panel_aggregation (pandas.tseries.tests.test_resample.TestTimeGrouper)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 1512, in test_panel_aggregation
    binagg = bingrouped.mean()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1145, in _python_agg_general
    for name, obj in self._iterate_slices():
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 3477, in _iterate_slices
    raise NotImplementedError("axis other than 0 is not supported")
NotImplementedError: axis other than 0 is not supported

======================================================================
ERROR: test_period_resample (pandas.tseries.tests.test_timeseries.TestTimeSeries)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_timeseries.py", line 2042, in test_period_resample
    result = s.to_period().resample('T', kind='period')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 92, in resample
    rs = self._resample_periods()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 357, in _resample_periods
    return grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2325, in aggregate
    return getattr(self, func_or_funcs)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1147, in _python_agg_general
    result, counts = self.grouper.agg_series(obj, f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1862, in agg_series
    return grouper.get_result()
  File "pandas\src\reduce.pyx", line 274, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:34306)
    raise
  File "pandas\src\reduce.pyx", line 249, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:33847)
    cached_ityp = self.ityp(islider.buf)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 184, in __new__
    ordinal, freq = cls._from_arraylike(data, freq, tz)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\period.py", line 250, in _from_arraylike
    raise ValueError('freq not specified and cannot be '
ValueError: freq not specified and cannot be inferred from first element

======================================================================
ERROR: pandas.tests.test_tseries.test_group_ohlc
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Miniconda\envs\pandas\lib\site-packages\nose\case.py", line 197, in runTest
    self.test(*self.arg)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tests\test_tseries.py", line 507, in test_group_ohlc
    _check('float32')
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tests\test_tseries.py", line 489, in _check
    func(out, counts, obj[:, None], labels)
  File "pandas\src\generated.pyx", line 7240, in pandas.algos.group_ohlc_float32 (pandas\algos.c:125372)
    def group_ohlc_float32(ndarray[float32_t, ndim=2] out,
ValueError: Buffer dtype mismatch, expected 'int64_t' but got 'long'

======================================================================
FAIL: test_resample_dup_index (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 454, in test_resample_dup_index
    result = df.resample('Q',axis=1)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\generic.py", line 3486, in resample
    return sampler.resample(self).__finalize__(self)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 92, in resample
    rs = self._resample_periods()
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\resample.py", line 357, in _resample_periods
    return grouped.aggregate(self._agg_method)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 2786, in aggregate
    return getattr(self, arg)(*args, **kwargs)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 738, in mean
    return self._python_agg_general(f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1147, in _python_agg_general
    result, counts = self.grouper.agg_series(obj, f)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\core\groupby.py", line 1862, in agg_series
    return grouper.get_result()
  File "pandas\src\reduce.pyx", line 238, in pandas.lib.SeriesBinGrouper.get_result (pandas\lib.c:33723)
    vslider = Slider(self.arr, self.dummy_arr)
  File "pandas\src\reduce.pyx", line 434, in pandas.lib.Slider.__init__ (pandas\lib.c:38534)
    assert(values.ndim == 1)
AssertionError

======================================================================
FAIL: test_resample_group_info (pandas.tseries.tests.test_resample.TestResample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 942, in test_resample_group_info
    assert_series_equal(left, right)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 865, in assert_series_equal
    assert_attr_equal('dtype', left, right)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 667, in assert_attr_equal
    left_attr, right_attr)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 757, in raise_assert_detail
    raise AssertionError(msg)
AssertionError: Attributes are different

Attribute "dtype" are different
[left]:  int32
[right]: int64

======================================================================
FAIL: test_aggregate_with_nat (pandas.tseries.tests.test_resample.TestTimeGrouper)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tseries\tests\test_resample.py", line 1607, in test_aggregate_with_nat
    assert_frame_equal(expected, dt_result)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 987, in assert_frame_equal
    obj='DataFrame.iloc[:, {0}]'.format(i))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 884, in assert_series_equal
    check_less_precise, obj='{0}'.format(obj))
  File "pandas\src\testing.pyx", line 58, in pandas._testing.assert_almost_equal (pandas\src\testing.c:3615)
    cpdef assert_almost_equal(a, b, bint check_less_precise=False,
  File "pandas\src\testing.pyx", line 147, in pandas._testing.assert_almost_equal (pandas\src\testing.c:2577)
    raise_assert_detail(obj, msg, lobj, robj)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 757, in raise_assert_detail
    raise AssertionError(msg)
AssertionError: DataFrame.iloc[:, 0] are different

DataFrame.iloc[:, 0] values are different (20.0 %)
[left]:  [1.2509771188, -0.314073359135, nan, -1.59805928254, -2.4494635435]
[right]: [1.2509771188, -0.314073359135, 0.0, -1.59805928254, -2.4494635435]

======================================================================
FAIL: test_groupby_with_timegrouper (pandas.tests.test_groupby.TestGroupBy)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tests\test_groupby.py", line 4020, in test_groupby_with_timegrouper
    assert_frame_equal(result1, expected)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 950, in assert_frame_equal
    '{0}, {1}'.format(right.shape[1], right.columns))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 757, in raise_assert_detail
    raise AssertionError(msg)
AssertionError: DataFrame are different

DataFrame shape (number of columns) are different
[left]:  2, Index([u'Buyer', u'Quantity'], dtype='object')
[right]: 1, Index([u'Quantity'], dtype='object')

======================================================================
FAIL: test_series_groupby_nunique (pandas.tests.test_groupby.TestGroupBy)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tests\test_groupby.py", line 1641, in test_series_groupby_nunique
    check_nunique(frame, ['jim'])
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tests\test_groupby.py", line 1631, in check_nunique
    assert_series_equal(left, right)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 865, in assert_series_equal
    assert_attr_equal('dtype', left, right)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 667, in assert_attr_equal
    left_attr, right_attr)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 757, in raise_assert_detail
    raise AssertionError(msg)
AssertionError: Attributes are different

Attribute "dtype" are different
[left]:  int32
[right]: int64

======================================================================
FAIL: test_timegrouper_with_reg_groups (pandas.tests.test_groupby.TestGroupBy)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\tests\test_groupby.py", line 4188, in test_timegrouper_with_reg_groups
    assert_frame_equal(result, expected)
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 950, in assert_frame_equal
    '{0}, {1}'.format(right.shape[1], right.columns))
  File "C:\Users\Jeff Reback\pandas\build\lib.win-amd64-2.7\pandas\util\testing.py", line 757, in raise_assert_detail
    raise AssertionError(msg)
AssertionError: DataFrame are different

DataFrame shape (number of columns) are different
[left]:  3, Index([u'Branch', u'Buyer', u'Quantity'], dtype='object')
[right]: 1, Index([u'Quantity'], dtype='object')

----------------------------------------------------------------------
Ran 8031 tests in 565.959s

FAILED (SKIP=622, errors=19, failures=6)

[pandas] C:\Users\Jeff Reback\pandas>
```

--- a/pandas/core/groupby.py
+++ b/pandas/core/groupby.py
@@ -1793,13 +1793,13 @@ class BinGrouper(BaseGrouper):
     @cache_readonly
     def group_info(self):
         ngroups = self.ngroups
-        obs_group_ids = np.arange(ngroups)
+        obs_group_ids = np.arange(ngroups, dtype='int64')
         rep = np.diff(np.r_[0, self.bins])
 
         if ngroups == len(self.bins):
-            comp_ids = np.repeat(np.arange(ngroups), rep)
+            comp_ids = np.repeat(np.arange(ngroups, dtype='int64'), rep)
         else:
-            comp_ids = np.repeat(np.r_[-1, np.arange(ngroups)], rep)
+            comp_ids = np.repeat(np.r_[-1, np.arange(ngroups, dtype='int64')], rep)
 
         return comp_ids, obs_group_ids, ngroups
 
@@ -2552,8 +2552,8 @@ class SeriesGroupBy(GroupBy):
 
         # group boundries are where group ids change
         # unique observations are where sorted values change
-        idx = np.r_[0, 1 + np.nonzero(ids[1:] != ids[:-1])[0]]
-        inc = np.r_[1, val[1:] != val[:-1]]
+        idx = com._ensure_int64(np.r_[0, 1 + np.nonzero(ids[1:] != ids[:-1])[0]])
+        inc = com._ensure_int64(np.r_[1, val[1:] != val[:-1]])
 
         # 1st item of each group is a new unique observation
         mask = isnull(val)

--- a/pandas/tseries/tests/test_resample.py
+++ b/pandas/tseries/tests/test_resample.py
@@ -919,7 +919,7 @@ class TestResample(tm.TestCase):
     def test_resample_group_info(self):  # GH10914
         for n, k in product((10000, 100000), (10, 100, 1000)):
             dr = date_range(start='2015-08-27', periods=n // 10, freq='T')
-            ts = Series(np.random.randint(0, n // k, n),
+            ts = Series(np.random.randint(0, n // k, n).astype('int64'),
                         index=np.random.choice(dr, n))
 
             left = ts.resample('30T', how='nunique')
@@ -1585,7 +1585,7 @@ class TestTimeGrouper(tm.TestCase):
         # check TimeGrouper's aggregation is identical as normal groupby
 
         n = 20
-        data = np.random.randn(n, 4)
+        data = np.random.randn(n, 4).astype('int64')
         normal_df = DataFrame(data, columns=['A', 'B', 'C', 'D'])
         normal_df['key'] = [1, 2, np.nan, 4, 5] * 4
 

